### PR TITLE
Create/Update - Set HasMany Associations on create and update

### PR DESCRIPTION
**Pull request by arnaudbesnier**

### CODE REVIEW

File: services/resource-updater.js [hunk 1]
```diff
@@ -9,6 +11,25 @@ function ResourceUpdater(model, params) {
...
+          return string.charAt(0).toUpperCase() + string.slice(1);
+        }
+
+        let promises = [];
+
+        if (model.associations) {
+          _.forOwn(model.associations, function(association, name) {
+            if (['HasOne', 'HasMany', 'BelongsToMany']
+              .indexOf(association.associationType) > -1) {
+              promises.push(record[`set${capitalize(name)}`](params[name]));
```
Comment (arnaudbesnier) [inline services/resource-updater.js]: There is an issue on hasOne relation update

File: services/resource-creator.js [hunk 2]
```diff
@@ -1,10 +1,30 @@
...
 var ResourceGetter = require('./resource-getter');
 
 function ResourceCreator(model, params) {
 
   this.perform = function () {
     return model.create(params)
+      .then(function(record) {
+        function capitalize(string) {
+          return string.charAt(0).toUpperCase() + string.slice(1);
+        }
```
Comment (SeyZ) [inline services/resource-creator.js]: use `_.capitalize`?

**Comment by SeyZ** [comment]
Congrats for the hasOne issue on sequelize :)

File: services/resource-updater.js [hunk 3]
```diff
@@ -9,6 +11,21 @@ function ResourceUpdater(model, params) {
         where: { id: params.id },
         individualHooks: true
       })
+      .then(() => model.findById(params.id))
```
Comment (SeyZ) [inline services/resource-updater.js]: Avoid ECMA6 on lianas?

**Pull request merged**